### DISCUSSION TITLE
Can Vuetify combined with Tailwindcss?

### DISCUSSION THREAD

**Discussion by iam-frankqiu** [vuetifyjs/vuetify#12765] [Ideas]
Because Tailwindcss can make tree-shaking to CSS. It can make Vuetify be smaller. So I think it's a good idea.

**Comment by velly** [comment]
This is a good idea. Now I just integrated them together with tailwind prefix.
If Vuetify could use tailwindcss in foundation dependency, that would be a cool thing to me.
